src/isEqual.ts

```diff
@@ -0,0 +1,76 @@
+import { purry } from "./purry";
+
+/**
+ * Performs a _native_ equality check between two values to. For primitive
```

Comment:
**nit:** this first sentence doesn't make sense, did you mean to write "to determine if they are equivalent"?

maybe we can even write "to determine if they are identical"?